doc/old_and_new_proposal.md

```diff
@@ -108,18 +178,304 @@ addition, every volume gets a "weight" (so far, based on the
+    Linux installation (partition id linux, swap, lvm or raid).
+  * Others delete mode: the same for all other partitions that don't fit into
+    the former two groups.
+  * Whether to resize Windows systems if needed
+
+### Creation of LVM structures
+
+Warning: this section describes some optional behaviours that are still not
+implemented in yast-storage-ng, although they would be very easy to add to the
+current codebase.
+
+As said before, using LVM doesn't make a big difference on how the proposal
+works. It simply allocates LVM logical volumes instead of partitions. To
+allocate such LVs, the proposal first needs to create (or reuse) a volume
+group that is big enough, which usually means creating one or several physical
+volumes.
+
+In the current implementation, the VG is created to perfectly fit the sizes of
+all the created LVs, with no extra unused space. It would be easy to add an
+configuration option to force the proposal to take more space than strictly
+needed by adding two possibilities: one for using all the available space
+(after deleting partitions according to the settings explained above) and
+another to use a fixed size for the VG (which must be, of course, equal or
+bigger than the sum of the max sizes of all the volumes).
+
+## Proposed settings for yast-storage-ng
+
+The redesign in the approach and code of the storage proposal explained above
+also deserves a revamp of the corresponding section in the `control.xml` file.
+This part of the document presents a possible alternative for the `partitioning`
+section of the control file. The goal is to provide more flexibility defining
+the products behavior and better access to all the features of the new storage
+proposal.
+
+In order to fully understand this section and its implications is important to
+have read and understood the above section titled "How the new proposal
+distributes the space".
+
+### General structure
+
+The new `partitioning` section will still contain the
+`root_subvolume_read_only`, `proposal_settings_editable`,
+`expert_partitioner_warning` and `use_separate_multipath_module` options
+described in this document's subsection "Other general options" at "Settings in
+yast-storage". These options will retain its exact form and meaning.
+
+In addition that, there will be two new subsections: `proposal` and `volumes`.
+
+### The `proposal` subsection
+
+The `proposal` subsection will be used to configure some general aspects of the
+storage proposal (referenced as "guided setup" in the UI) and will contain the
+following options.
+
+  * `default_lvm`. Whether LVM should be used by default.
+  * `default_encrypt`. Whether encryption should be used by default.
+  * `default_windows_delete`. Default value for the automatic delete mode for
+    Windows partitions. It can be `none`, `all` or `ondemand`. For more
+    information, see the description of the new proposal above.
+  * `default_linux_delete`. Default value for the automatic delete mode for
```

Comment:
Only to clarify, I would say explicitly that the options are `none`, `all` or `ondemand` again.